components/UserProfilePanels/UserSettingsPanel.js

```diff
@@ -206,29 +206,29 @@ class GroupSettingsPanel extends Component {
                 value={bio} />
             </fieldset>
 
-            {/*<fieldset>
+            <fieldset>
               <label htmlFor="name">
```

Comment:
We should update the `htmlFor` to be the id of the input that it's labelling (`email`), which negates the need for `aria-label` on the input.

Reply:
ah right. Yeah I wasn't EXACTLY paying attention to this XD